### PR TITLE
Allow config of filter params; add missing tests

### PR DESCRIPTION
**Pull request by ravigadad**

### CODE REVIEW

File: lib/rory/request_parameter_logger.rb [hunk 1]
```diff
@@ -3,10 +3,10 @@
 module Rory
   class RequestParameterLogger
 
-    def initialize(app, logger=nil, filters=[:password])
+    def initialize(app, logger = nil, options = {})
       @app = app
       @logger = logger
-      @filters = filters
+      @filters = options.fetch(:filters, [])
```
Comment (michaelirey) [inline lib/rory/request_parameter_logger.rb]: Curious, why the removal of :password as a default filter?

Comment (michaelirey) [inline lib/rory/request_parameter_logger.rb]: Nevermind, I see it in lib/rory/application.rb

File: spec/fixture_app/config/application.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 module Fixture
   class Application < Rory::Application
     turn_off_request_logging!
+    filter_parameters :orcas, :noodles
```
Comment (michaelirey) [inline spec/fixture_app/config/application.rb]: Is this a convention you prefer? I would of passed in an array parameters to filter, and they are going to end up in an array anyways. To me it is more clear that I am passing in a collection of things vs going to look up and see the method definition is doing with the first, second, third, etc... params.

File: spec/lib/rory/application_spec.rb [hunk 3]
```diff
@@ -90,7 +96,7 @@ class RootlessApp < Rory::Application; end
   end
 
   describe ".logger" do
-    it "reutrns a logger" do
+    it "returns a logger" do
```
Comment (michaelirey) [inline spec/lib/rory/application_spec.rb]: nice

File: lib/rory/application.rb [hunk 4]
```diff
@@ -118,15 +118,28 @@ def request_logging_on?
...
+      reset_stack
       @request_logging = false
     end
 
+    def parameters_to_filter
+      @parameters_to_filter || [:password]
+    end
+
+    def filter_parameters(*params)
+      reset_stack
```
Comment (michaelirey) [inline lib/rory/application.rb]: nice

**Comment by michaelirey** [comment]
+17 for the additional test coverage

File: spec/fixture_app/config/application.rb (hunk 2 above)
Comment (ravigadad) [inline spec/fixture_app/config/application.rb]: That's a good question.  I do like this convention, yes, but I'm not stuck on it, and you make a good point about what is expected.  I myself like the idea that (with named parameters) we're moving towards an expectation that multiple unnamed parameters are a collection.

Comment (michaelirey) [inline spec/fixture_app/config/application.rb]: Ok, its good to hear your thoughts on this. I am not saying what I suggested was best, just different and was curious to hear your thinking around the approach you took.

**Pull request merged**